amenity-points.mss

```diff
@@ -852,7 +854,8 @@
     marker-fill: @airtransport;
   }
 
-  [feature = 'aeroway_aerodrome'][zoom >= 10][zoom < 14] {
+  [feature = 'aeroway_aerodrome'][zoom >= 14][way_pixels < @area-too-large-to-label-size-in-pixels],
+  [feature = 'aeroway_aerodrome'][zoom >= 10][way_pixels > 5][way_pixels < @area-too-large-to-label-size-in-pixels] {
```

Comment:
But it should fairly easy to modify - and I preferred to give too low value rather than too big.